Loop/View Controllers/StatusTableViewController.swift

```diff
@@ -1911,6 +1911,12 @@ final class StatusTableViewController: LoopChartsTableViewController {
         actionSheet.addAction(UIAlertAction(title: "Delete CGM Manager", style: .destructive) { _ in
             self.deviceManager.cgmManager?.delete() { }
         })
+        
+        actionSheet.addAction(UIAlertAction(title: "Delete Pump Manager", style: .destructive) { _ in
```

Comment:
I think this allows us to remove the delete functionality that I moved to the hidden Palmtree pump manager page since it's only accessible for the demo?